MetadataExtractor.PowerShell/ExtractMetadataCmdlet.cs

```diff
@@ -23,7 +24,7 @@ protected override void ProcessRecord()
         {
             base.ProcessRecord();
 
-            WriteVerbose($"Extracting metadata from file: {FilePath}");
+            WriteVerbose(string.Format(CultureInfo.InvariantCulture, "Extracting metadata from file: {0}", FilePath));
```

Comment:
Were these transformations performed manually or via automation?

Reply:
Hand made with ♥. I don't know any tool for that.

Reply:
I thought perhaps the analyzer came with a codefix.

Reply:
Great idea!